File: Beecrowd/Beginner/2003 - 2176/2143 - The Return of Radar/C++/2143_TheReturnofRadar.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    int testCases;

    while (cin >> testCases) {
        if (testCases == 0) break;

        while (testCases--) {
            int people;
            cin >> people;

            if (people % 2 == 0) {
                cout << (people - 2) * 2 + 2 << endl;
            } else cout << (people - 1) * 2 + 1 << endl;
        }
    }

    return 0;
}

File: Beecrowd/Beginner/2003 - 2176/2143 - The Return of Radar/Python/2143_TheReturnofRadar.py
while True:
    testCases = int(input())
    if testCases == 0:
        break

    for _ in range(testCases):
        people = int(input())

        if people % 2 == 0:
            print((people - 2) * 2 + 2)
        else:
            print((people - 1) * 2 + 1)
